packages/expo-device/ios/UIDevice.swift

```diff
@@ -0,0 +1,199 @@
+            case "iPad6,7", "iPad6,8":                            return DeviceMapping(modelName: "iPad Pro (12.9-inch) (1st generation)", deviceYearClass: 2015)
+            case "iPad7,1", "iPad7,2":                            return DeviceMapping(modelName: "iPad Pro (12.9-inch) (2nd generation)", deviceYearClass: 2017)
+            case "iPad8,5", "iPad8,6", "iPad8,7", "iPad8,8":      return DeviceMapping(modelName: "iPad Pro (12.9-inch) (3rd generation)", deviceYearClass: 2018)
+            case "iPad8,11", "iPad8,12":                          return DeviceMapping(modelName: "iPad Pro (12.9-inch) (4th generation)", deviceYearClass: 2020)
+            case "iPad13,8", "iPad13,9", "iPad13,10", "iPad13,11": return DeviceMapping(modelName: "iPad Pro (12.9-inch) (5th generation)", deviceYearClass: 2021)
+            case "AppleTV5,3":                                    return DeviceMapping(modelName: "Apple TV HD (4th Generation, Siri)", deviceYearClass: 2015)
+            case "AppleTV6,2":                                    return DeviceMapping(modelName: "Apple TV 4K", deviceYearClass: 2017)
+            case "i386", "x86_64", "arm64":                       return DeviceMapping(modelName: "Simulator iOS", deviceYearClass: nil)
+            default:                                              return DeviceMapping(modelName: identifier, deviceYearClass: nil)
+            }
+            #elseif os(tvOS)
+            switch identifier {
+            case "AppleTV5,3":  return DeviceMapping(modelName: "Apple TV HD (4th Generation, Siri)", deviceYearClass: 2015)
+            case "AppleTV6,2":  return DeviceMapping(modelName: "Apple TV 4K", deviceYearClass: 2017)
+            case "i386", "x86_64": return DeviceMapping(modelName: "Simulator tvOS", deviceYearClass: nil)
+            default: return identifier
+            }
+            #endif
+        }
+
+        return mapToDevice(identifier: modelIdentifier)
+    }()
+
+    // Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
+      var isSimulator: Bool {
+          return TARGET_OS_SIMULATOR != 0
+      }
+
+      var isJailBroken: Bool {
+          get {
+              if UIDevice.current.isSimulator {
+                return false
+              }
+              if JailBreakHelper.hasCydiaInstalled() {
+                return true
+              }
+              if JailBreakHelper.isContainsSuspiciousApps() {
+                return true
+              }
+              if JailBreakHelper.isSuspiciousSystemPathsExists() {
+                return true
+              }
+              return JailBreakHelper.canEditSystemFiles()
+          }
+      }
+}
+
+// Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
+private struct JailBreakHelper {
```

Comment:
I think it's commonly spelled as `Jailbreak`, not `JailBreak`
```suggestion
private struct JailbreakHelper {
```